<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Uninstall</class>
 <widget class="QDockWidget" name="Uninstall">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Uninstall</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="layout_package_list">
      <item>
       <layout class="QHBoxLayout" name="layout_search">
        <item>
         <widget class="QLabel" name="label_search">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Search:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="entry_search"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListView" name="packages_list"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="button_uninstall">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Uninstall</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="9"/>
 <resources/>
 <connections/>
</ui>
